src/components/FidelityBondDevOnly.tsx

```diff
@@ -0,0 +1,350 @@
+import React, { useEffect, useMemo, useState } from 'react'
+import * as rb from 'react-bootstrap'
+import { Trans, useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import { useCurrentWallet, useReloadCurrentWalletInfo, Utxos } from '../context/WalletContext'
+import { CopyButtonWithConfirmation } from '../components/CopyButton'
+import { isFeatureEnabled } from '../constants/features'
+
+// @ts-ignore
+import PageTitle from './PageTitle'
+// @ts-ignore
+import DisplayUTXOs from './DisplayUTXOs'
+
+import { routes } from '../constants/routes'
+import * as Api from '../libs/JmWalletApi'
+import styles from './FidelityBond.module.css'
+
+type AlertWithMessage = rb.AlertProps & { message: string }
+
+const dateToLockdate = (date: Date): Api.Lockdate =>
+  `${date.getUTCFullYear()}-${date.getUTCMonth() >= 9 ? '' : '0'}${1 + date.getUTCMonth()}` as Api.Lockdate
+
+const lockdateToTimestamp = (lockdate: Api.Lockdate): number => {
+  const split = lockdate.split('-')
+  return Date.UTC(parseInt(split[0], 10), parseInt(split[1], 10) - 1, 1, 0, 0, 0)
```

Comment:
```suggestion
  return Date.UTC(parseInt(split[0], 10), parseInt(split[1], 10) - 1, 1)
```

The time parameters are 0 [per default](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC#syntax). the day is 1 per default as well but I'd say that's fine to leave in for purposes of documentation.

Reply:
Thought to have the actual day time related values as well for "purposes of documentation", but that is also fine with me. Removed!